src/components/NavBar/index.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <nav class="navbar background-grey">
+    <div class="container">
+      <div class="navbar-brand logo dp-flex f-drt-column jtf-ct-center">
+        VueTube
+      </div>
+      <div class="navbar-start">
+        <div class="navbar-item">
+          <b-input class="pd-10px column" placeholder="Search" v-model="keyword" @keyup.enter.native="search(keyword)"></b-input>
+        </div>
+      </div>
+    </div>
+  </nav>
+</template>
+
+<script>
+import { mapActions } from 'vuex'
+
+export default {
+  data () {
+    return {
+      keyword: ''
+    }
+  },
+  methods: {
+    ...mapActions([
+      'search'
+    ])
+  },
+  created () {
+    this.search('')
+  }
+}
+</script>
+
+<style scoped>
+.logo {
+  font-size: 25px;
+  font-weight: bold;
+  color: white;
```

Comment:
Maybe `color` should be `#f00` , What do you think ?

Reply:
I used white color because I would like to make YouTube parody 😎.

But it's okay that you prefer `#f00` color. I appreciate both of them.